### PR TITLE
fix: modify getting started docs

### PR DESCRIPTION
**Pull request by CarolineGard**
Clarify the getting-started docs, by separating the `configuration and setups` block. 
Added `Alternative 1: Add uno.config file` and `Alternative 2: Include Uno directly in the build setup` instead of combining both in the same code example with comments. 

When combining them in the same code example, this confused consumers to think they were supposed to add both examples when setting up warp. 

+ some cleanup

### CODE REVIEW

**Review by siljec — APPROVED** [review]
Nice 🎉

File: docs/getting-started/developers/index.md [hunk 1]
```diff
@@ -51,22 +65,43 @@ export default defineConfig({
 });
 ```
 
-#### Adding Uno to your build tool
+#### Add Uno to your build tool
```
Comment (BalbinaK) [inline docs/getting-started/developers/index.md]: Maybe we should align with how the setup was explained in the [css docs](https://github.com/warp-ds/css-docs/pull/59/files). There we also suggested not to include uno in the build step after all due to possible issues unocss may have when used with VSCode plugins for autocompletion. Those will work best if a `uno.config.js` file is found according to their documentation.

File: docs/getting-started/developers/index.md [hunk 2]
```diff
@@ -12,32 +12,46 @@ If you are migrating from Fabric to Warp, please visit the [Migration page](/mig
 
 This is the initial version of Warp and is currently a work in progress. Any elements, components or inline styled/custom CSS that are not implemented using Fabric, is required to be managed in order to make them themeable.
 
+## Before getting started
+When migrating to Warp, this may affect or break the styling for other teams that exists on the same page as your project. **Make sure to communicate and align with affected teams before starting the migration process.**
```
Comment (BalbinaK) [inline docs/getting-started/developers/index.md]: Should this be part of the Migration to Warp page? We are linking those who are migrating to Warp from Fabric in the previous header and they might miss this important message. Wdyt?

File: docs/getting-started/developers/index.md [hunk 3]
```diff
@@ -51,22 +65,43 @@ export default defineConfig({
...
-By default, UnoCSS will automatically look for uno.config.{js,ts,mjs,mts} or unocss.config.{js,ts,mjs,mts} in the root directory of your project. You can also specify the config file manually and in that case you won't need a separate uno config file. Like in this example for Vite:
+Then add Uno to your build tool
 
+> vite.config.js
 ```ts
-// vite.config.js
 import { defineConfig } from 'vite';
 import UnoCSS from 'unocss/vite';
 import { presetWarp } from '@warp-ds/uno';
 import { classes } from '@warp-ds/component-classes/classes';
```
Comment (BalbinaK) [inline docs/getting-started/developers/index.md]: We should remove these imports when displaying Alternative 1.

**Review by BalbinaK — APPROVED** [review]

**Pull request merged**